lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import Tab from './Tab';
+import PropTypes from 'prop-types';
+
 const Tabs = props => {
   return (
     <div className="tabs">
       <div className="topics">
         <span className="title">TRENDING TOPICS:</span>
         {/* map over the tabs provided on your props, create a new Tab component for each one.
             give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/}
+        {props.tabs.map((tab,index) => <Tab key={index} tab={tab} selectTabHandler={props.selectTabHandler} selectedTab={props.selectedTab} />)}
```

Comment:
Using an index from the map method for a key is never a good idea. In this instance it works because the tabs never change, but if we dynamically rendered tabs you would get unwanted behavior